### DISCUSSION TITLE
[Feb 08] URL to file attachment

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1273] [Discord Threads]
> **moosefist** _queried at_ ```2022-02-07T18:19:05.319000+00:00```  
## If i upload spreadsheet that has a URL that links to a contact image, is there a way to add that as a file attachment?  
  
> **o1lab** _commented at_ ```2022-02-07T18:20:34.905000+00:00```  
#### It's not possible unfortunately, but we can take this as a feature request.  
  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/940310718081818634) 
 
 - - - -

#### Join [NocoDB's](http://www.nocodb.com/)  community
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by tracure1337** [comment]
This is a good idea. I wanted to add status badges as fetched url from an API.
